Library/Homebrew/rubocops/cask/array_alphabetization.rb

```diff
@@ -0,0 +1,38 @@
+# typed: true
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Cask
+      class ArrayAlphabetization < Base
+        extend AutoCorrector
+
+        def on_send(node)
+          return if node.method_name != :zap
```

Comment:
I love this! My only suggestion would be to enforce this over both `:zap` and `:uninstall` arrays, since the methods are identical.